### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Oleksandr147**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
```
Comment (Oleksandr147) [inline app/main.py:18]: rename to 'character'

File: app/main.py [hunk 2]
```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
...
+        if char.lower() == letter.lower():
+            counter += 1
```
Comment (Oleksandr147) [inline app/main.py:20]: add a line

File: app/main.py [hunk 3]
```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for character in phrase:
+        if character.lower() == letter.lower():
+            counter += 1
```
Comment (EdAlekseiev) [inline app/main.py:20]: Write this in one line

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Comment by Oleksandr147** [comment]
Already created new pull request, please check

**Pull request closed without merging**